.github/workflows/ci_benchmark.yml

```diff
@@ -87,6 +87,11 @@ jobs:
               | sed "/Traceback \|failed$\|PERFORMANCE DECREASED/ s/^/::error::/" \
               | tee benchmarks.log
 
+          # Error out the action
+          if grep "Traceback \|failed\|PERFORMANCE DECREASED" benchmarks.log > /dev/null ; then
```

Comment:
Can just use `grep -q ...` and avoid directing anything to `/dev/null`

Reply:
Yeah that should work fine too!